configs/bundle_config.yaml

```diff
@@ -357,11 +362,24 @@ databundles:
     category: cutouts
     destination: "cutouts"
     urls:
+      zenodo: https://sandbox.zenodo.org/records/418694/files/bundle_cutouts_westasia.zip?download=1
       gdrive: https://drive.google.com/file/d/1as-fZS4Z70CfDf3bR4i9t9S5-5pBqURP/view?usp=sharing
     output: [cutouts/cutout-2013-era5.nc]
     disable_by_opt:
       build_cutout: [all]
 
+  # cutouts bundle of the cutouts folder for North Eurasia
+  # Size about 19.6 GB (zipped)
+  bundle_cutouts_northernasia:
+    countries: ["LT", "LV", "EE", "BY", "RU", "KZ", "MN", "GE"]
```

Comment:
May it be a good idea to revise into `bundle_cutouts_northeurasia` (it covers Nordic regions of both Europe and Asia)? Another option could be `bundle_cutouts_subarctic`

Reply:
Northeurasia?

We don't include Canada, so subartic looks to me partial

Plus, I'll add a release note

Reply:
Yeah, agree! I think northeurasia sounds good 🙂